test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -69,15 +70,15 @@ estimate: 90m
 
     This updates the per minute rate limit to 694
 
-4.  Modify the `rate-limit-alerts` to allow alerts to fire on a per minute basis:
+4.  Modify the `rate-limit-alerts` to allow alerts to fire on a per minute basis. Note the original values to be able to revert back once finished:
 
     ```shell script
     oc patch configmap rate-limit-alerts -n redhat-rhoam-operator -p '"data": { "alerts": "{\n  \"api-usage-alert-level1\": {\n    \"type\": \"Threshold\",\n    \"level\": \"info\",\n    \"ruleName\": \"RHOAMApiUsageLevel1ThresholdExceeded\",\n    \"period\": \"1m\",\n    \"threshold\": {\n      \"minRate\": \"80%\",\n      \"maxRate\": \"90%\"\n    }\n  },\n  \"api-usage-alert-level2\": {\n    \"type\": \"Threshold\",\n    \"level\": \"info\",\n    \"ruleName\": \"RHOAMApiUsageLevel2ThresholdExceeded\",\n    \"period\": \"1m\",\n    \"threshold\": {\n      \"minRate\": \"90%\",\n      \"maxRate\": \"95%\"\n    }\n  },\n  \"api-usage-alert-level3\": {\n    \"type\": \"Threshold\",\n    \"level\": \"info\",\n    \"ruleName\": \"RHOAMApiUsageLevel3ThresholdExceeded\",\n    \"period\": \"1m\",\n    \"threshold\": {\n      \"minRate\": \"95%\"\n    }\n  },\n  \"rate-limit-spike\": {\n    \"type\": \"Spike\",\n    \"level\": \"warning\",\n    \"ruleName\": \"RHOAMApiUsageOverLimit\",\n    \"period\": \"30m\"\n  }\n}"}'
     ```
 
 5.  Patch the `rhoam` CR to specify BU, SRE and Customer email addresses:
 
-    _NOTE:_ Replace `<rh_username>` references in the below commands with a valid Red Hat username. For example: `pamccart+BU@redhat.com`
+    _NOTE:_ Replace `<rh_username>` references in the below commands with a valid Red Hat username. For example: `pamccart+BU@redhat.com`. Note the original values to be able to revert back once finished.
```

Comment:
should [the changes on these lines](https://github.com/integr8ly/integreatly-operator/pull/1927/files#diff-9925ccbb89aaa85d0090bc55d19296fda08506de84e32faba7bae2eb95012278R73-R81) be reverted as well?

Reply:
Ah, right, good catch, completely forgot on those two lines.

Reply:
Addressed. @psturc Could you please have yet another look?